src/Engine/GameInstance.java

```diff
@@ -56,6 +56,11 @@ public void setCursorLocation(int x, int y)
     cursorX = x;
     cursorY = y;
   }
+  public void setCursorLocation(XYCoord coord)
```

Comment:
Whuh-oh. I also added this function in the `resupply` branch (I just added it before the overload instead of after like you did). Which might mean it won't conflict if we merge, but will cause build errors. Probably doesn't really matter for now.